main.cpp

```diff
@@ -107,9 +107,53 @@ struct CarWash
     you should be able to deduce the return type of those functions based on their usage in Person::run()
     You'll need to insert the Person struct from the video in the space below.
  */
+struct Person
+{
+    int age;
+    int height;
+    float hairLength;
+    float GPA;
+    unsigned int SATScore;
+    int distanceTraveled;
+
+    struct Foot
+    {
+        void stepForward();
+        int stepSize();
+    };
 
+    void run(int howFast, bool startWithLeftFoot);
 
+    Foot leftFoot;
+    Foot rightFoot;
+};
+
+void Person::run(int howFast, bool startWithLeftFoot)
+{
+    if (startWithLeftFoot == true)
+    {
+        leftFoot.stepForward();
+        rightFoot.stepForward();
+    } 
+    else 
+    {
+        rightFoot.stepForward();
+        leftFoot.stepForward();
+    }
+    distanceTraveled += (leftFoot.stepSize() + rightFoot.stepSize()) * howFast;
+}
+
+// Not quite sure if this is the right syntax and if it's better practice to put this here or in the struct itself
+void Person::Foot::stepForward()
+{
+    std::cout << "Steps forward" << std::endl;
+}
 
+int Person::Foot::stepSize()
+{
+    // I couldn't quite figure out how to access the distanceTraveled member variable. Wanted to modify that variable as part of this function, but haven't wrapped my head around inheritance yet I think, so I'll just simply return an int for now
+    return 1;
+}
```

Comment:
This implementation is fine.
`distanceTraveled` is part of the `Person` class.  it's not part of the `Foot` class.
That's why you can't access it.